local.c

```diff
@@ -1181,6 +1181,14 @@ static int read_device_name(struct iio_device *dev)
 		return 0;
 }
 
+static int cmp_iio_device_attr(const void *p1, const void *p2)
```

Comment:
Most functions in this file are prefixed local_, would be good to continue this pattern.

E.g. local_compare_device_attr().